assets/app.js

```diff
@@ -338,6 +338,39 @@ new Vue({
         }
     },
     computed: {
+        validateCacheMaterial: function () {
```

Comment:
This is not how you use **computed** properties, you named, coded and use it like a normal **method**
please move.